gulpfile.js/tasks/rev/rev-css.js

```diff
@@ -1,6 +1,6 @@
 var gulp        = require('gulp')
 var rev         = require('gulp-rev')
-var revNapkin   = require('gulp-rev-napkin')
+var revDel      = require('gulp-rev-delete-original')
```

Comment:
Particular reason for the swap?

Reply:
napkin hasnt been updated in 3 years and delete-original has been updated in the last year. napkin was still reliant on gulp-util that was throwing errors, so i didn't really think the maintainer was going to push up anything anytime soon to resolve this